### ISSUE TITLE
Inserimento di "Niente" non immediato

### ISSUE THREAD

**Issue by ag7** [develersrl/app-pranzi#5]
A differenza delle altre opzioni (P, S...) che devono essere confermate premendo "Invia", se l'utente seleziona "Niente" parte subito la POST verso il server.

Secondo me andrebbe anche qui confermata l'azione premendo "Invia".
Potremmo deselezionare tutte le altre (eventuali) opzioni attive nel momento in cui si preme "Niente".

**Comment by tommyblue** [comment]
+1

**Comment by asterix24** [comment]
+1

**Comment by lvillani** [comment]
Secondo me é troppo complicato. Anche le mie applicazioni native hanno il pulsante "Niente" che fa direttamente la POST. Si tratta di uno shortcut che trovo sensato.

**Comment by lmancini** [comment]
Concordo: se schiaccio "Niente", non c'è nulla da confermare, può partire direttamente la post.

**Comment by ag7** [comment]
Ragazzi upvotate o downvotate la issue con 👍  o 👎  sul primo commento.
Per ora siamo 3 a 2 per il "Niente" non immediato.

**Comment by tommyblue** [comment]
Pensandoci bene, se la UI rende chiaro cosa è stato fatto non c'è bisogno. Sarebbe magari utile un "Annulla", in stile GMail quando elimini una mail, per rimettere il valore precedente
